src/tools/compiletest/src/runtest.rs

```diff
@@ -3458,6 +3458,21 @@ impl<'test> TestCx<'test> {
         let rmake_out_dir = base_dir.join("rmake_out");
         create_dir_all(&rmake_out_dir).unwrap();
 
+        // Copy all input files (apart from rmake.rs) to the temporary directory,
+        // so that the input directory structure from `tests/run-make/<test>` is mirrored
+        // to the `rmake_out` directory.
+        for path in walkdir::WalkDir::new(&self.testpaths.file).min_depth(1) {
+            let path = path.unwrap().path().to_path_buf();
+            if path.file_name().map(|s| s != OsStr::new("rmake.rs")).unwrap_or(false) {
```

Comment:
Suggestions:

- `s != "rmake.rs"`, I think there's an impl for `PartialEq` between `&str` and `OsStr`?
- `file_name().map().unwrap_or(false)` can be written as `file_name().is_some_and(|n| n != "rmake.rs")`

Reply:
TIL about that `PartialEq` impl, thanks!